[MP-5726] Merchant Portal Design Clean Up (#535)
spryker/spryker-docs: docs/marketplace/user/merchant-portal-user-guides/202108.0/profile/editing-merchants-profile-details.md

| Imprint | In this field, you can specify imprint information. |   |
| Data Privacy | In this field, you can specify the data privacy statement. |   |

#### Shop Status

| ATTRIBUTE | DESCRIPTION | REQUIRED? |
|-|-|-|

> | Your Store is online | By selecting this option, you can define whether the merchant store (with the profile, merchant products and offers) is displayed in the Marketplace Storefront or not. |   |

Reviewer: ```suggestion
| Your Store is online | By selecting this option, you can define whether the merchant store (with the profile, merchant products and offers) should be displayed on the Marketplace Storefront. |   |
```
